### PR TITLE
List files missing tag needed to order content

### PR DESCRIPTION
**Pull request by bryanbrattlof**
This should fix: #2467

### CODE REVIEW

File: pelican/utils.py [hunk 1]
```diff
@@ -763,9 +763,18 @@ def order_content(content_list, order_by='slug'):
                     content_list.sort(key=attrgetter(order_by),
                                       reverse=order_reversed)
                 except AttributeError:
-                    logger.warning(
-                        'There is no "%s" attribute in the item '
-                        'metadata. Defaulting to slug order.', order_by)
+                    for content in content_list:
+                        if getattr(content, order_by, None) is not None:
```
Comment (avaris) [inline pelican/utils.py]: This assumes `getattr(content, order_by)` will not be `None`.

**Comment by bryanbrattlof** [comment]
Thanks @avaris, I was trying to avoid having to indent so much, and assumed that everything we sort by would return something other than `None`.

I changed to a `try/except AttributeError` to get rid of the assumption.

**Comment by avaris** [comment]
> assumed that everything we sort by would return something other than None.

Generally true, for the core at least, but with plugins and such you never know :).

**Comment by justinmayer** [comment]
Many thanks to @bryanbrattlof for the enhancement and to @avaris for the review. Sincere apologies for the long delay in getting to this, folks.

**Pull request merged**